docs/guide/browser-compatibility.md

```diff
@@ -89,6 +89,12 @@ Vue CLI uses two environment variables to communicate this:
 **Important:** These variables are only accessible when/after `chainWebpack()` and `configureWebpack()` functions are evaluated, (so not directly in the `vue.config.js` module's root scope). That means it's also available in the postcss config file.
 :::
 
+::: warning Caveat: Adjusting webpack plugins
+Some Plugins, i.e. `html-webpack-plugin`, `preload-plugin` etc. are only included in the config for modern mode. Trying to tap into their options in the legacy config can s an error as the plugins don't exist.
```

Comment:
`can s an error` - what does `s` stands for here? 😅

Reply:
Good question, I can't exactly recall but I think it would have been something like "**throw** an error"